src/ui/EditTask.svelte

```diff
@@ -35,59 +36,44 @@
     let parsedRecurrence: string = '';
     let parsedDone: string = '';
 
-    $: {
-        if (!editableTask.startDate) {
-            parsedStartDate = '<i>no start date</>';
-        } else {
-            const parsed = chrono.parseDate(
-                editableTask.startDate,
-                new Date(),
-                {
-                    forwardDate: true,
-                },
-            );
-            if (parsed !== null) {
-                parsedStartDate = window.moment(parsed).format('YYYY-MM-DD');
-            } else {
-                parsedStartDate = '<i>invalid start date</i>';
-            }
+    function parseDate(
+        type: 'start' | 'scheduled' | 'due' | 'done',
+        date: string,
+        forwardDate: Date | undefined = undefined,
+    ): string {
+        if (!date) {
+            return `<i>no ${type} date</>`;
```

Comment:
I think there is an i missing in the closing tag? I know it was missing in some of the original versions, so unsure if it matters or not.

Reply:
Yes it seems to be also missing in the original: https://github.com/schemar/obsidian-tasks/blob/44262750254dd8571dbea5ecef048cbbee58e727/src/ui/EditTask.svelte#L40

But I would guess it should be there, Svelte seems to be smart and closed it automatically